src/Controls/src/Core/Element/Element.cs

```diff
@@ -333,30 +333,32 @@ internal Element ParentOverride
 		}
 
 		WeakReference<Element> _realParent;
-		/// <summary>For internal use by .NET MAUI.</summary>
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public Element RealParent
+		Element GetRealParent(bool logWarningIfParentHasBeenCollected = true)
 		{
-			get
+			if (_realParent is null)
```

Comment:
Inline with @MartyIX 's comment can we just do all this in an if statement?

something like

```C#
if (_realParent is WeakRef wr &&  wr.TryGetTarget(out var parent))
```